EXILED/Exiled.API/Features/Roles/FpcRole.cs

```diff
@@ -243,6 +243,42 @@ public bool IsNoclipEnabled
             set => Owner.ReferenceHub.playerStats.GetModule<AdminFlagsStat>().SetFlag(AdminFlags.Noclip, value);
         }
 
+        /// <summary>
+        /// Gets or sets <see cref="API.Features.Ragdoll"/> for this role.
+        /// </summary>
+        public Ragdoll Ragdoll
+        {
+            get => Ragdoll.Get(FirstPersonController.Ragdoll);
+            set => FirstPersonController.Ragdoll = value.Base;
+        }
```

Comment:
That Ragdoll used to be prefab for spawning real ragdoll, it shouldnt be in the Radoll list, and would be a bit innaccuracy to give possibility to change radgoll prefab